OracleWebLogic/dockerfiles/14.1.1.0/Dockerfile.developer-11

```diff
@@ -0,0 +1,109 @@
+#Copyright (c) 2014-2019 Oracle and/or its affiliates. All rights reserved.
+#
+#Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for WebLogic 14.1.10 Quick Install Distro
+#
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+# (1) fmw_14.1.10.0_wls_quick_Disk1_1of1.zip
+#     Download the Developer Quick installer from http://www.oracle.com/technetwork/middleware/weblogic/downloads/wls-for-dev-1703574.html
+#
+# (2) server-jre-8uXX-linux-x64.tar.gz
+#     Download from http://www.oracle.com/technetwork/java/javase/downloads/server-jre8-downloads-2133154.html
+#
+# HOW TO BUILD THIS IMAGE
+# -----------------------
+# Put all downloaded files in the same directory as this Dockerfile
+# Run:
+#      $ docker build -f Dockerfile.developer -t oracle/weblogic:14.1.10-developer .
+#
+# IMPORTANT
+# ---------
+# The resulting image of this Dockerfile contains a WLS Empty Domain.
+#
+# From the OIracle Docker GitHub Project
+# --------------------------------------
+FROM oracle/jdk:11 as builder
+
+# Maintainer
+# ----------
+MAINTAINER Monica Riccelli <monica.riccelli@oracle.com>
+
+# Common environment variables required for this build (do NOT change)
+# --------------------------------------------------------------------
+ENV ORACLE_HOME=/u01/oracle \
+    USER_MEM_ARGS="-Djava.security.egd=file:/dev/./urandom" \
+    PATH=$PATH:${JAVA_HOME}/bin:/u01/oracle/oracle_common/common/bin:/u01/oracle/wlserver/common/bin
+
+# Setup filesystem and oracle user
+# Adjust file permissions, go to /u01 as user 'oracle' to proceed with WLS installation
+# ------------------------------------------------------------
+RUN mkdir -p /u01 && \
+    chmod a+xr /u01 && \
+    useradd -b /u01 -d /u01/oracle -m -s /bin/bash oracle
+
+# Environment variables required for this build (do NOT change)
+# -------------------------------------------------------------
+# ENV FMW_PKG=fmw_14.1.1.0.0_wls_quick_Disk1_1of1.zip \
```

Comment:
You have commented out FMW_PKG.  Is this because of local testing?  How does a customer receive the 14.1.1.0 download package?

Reply:
ZIP installers are not available just the JARs.  When ZIP installers are available I will un-comment those lines.  Also the download files and the Checksums do not apply until we have installer ZIP files.